examples/README.md

```diff
@@ -1,4 +1,4 @@
-# Hot Function Example
+# Hot JSON Function Example
```

Comment:
it's unclear why this change is here, the example provided gives `--format http`. since that's tested, I think it's fine to leave that as is and undo this edit. since the example will work with http and json since it uses the fdk, i'm not sure it really matters to highlight the specificity anymore

Reply:
Fixed. Decided to add the link to the fn formats doc which is quite useful in understanding those.